Sentinel.podspec

```diff
@@ -25,6 +25,7 @@ Pod::Spec.new do |s|
   s.platform = :ios
   s.swift_version = '5.0'
   s.ios.deployment_target = '14.0'
+  s.osx.deployment_target = '12.0'
```

Comment:
Not sure if this was added, but please update the `README.md` with that information.

Reply:
I would go through the readme again once we wrap up everything since there were quite a lot of changes, and I wouldn't want to do it multiple times if something changes before we release